source/examples/queue-parallel.cpp

```diff
@@ -0,0 +1,34 @@
+// SPDX-FileCopyrightText: 2020 The Khronos Group Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <CL/sycl.hpp>
```

Comment:
To update to SYCL 2020...

Reply:
Not sure what you mean by this comment, could you clarify @keryell?

Reply:
This is
```c++
#include <sycl/sycl.hpp>
```
in SYCL 2020.

Reply:
Ah, understood. I'll fix that.